migration/sqls/000005_add_pull_consumer.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `consumer`
+ADD COLUMN `type` varchar(4) NOT NULL DEFAULT 'push';
```

Comment:
```
ALTER TABLE `consumer`
ADD COLUMN `type` ENUM('push', 'pull') NOT NULL
DEFAULT ('push');
```
I tried to use this migration with `ENUM` here. It works when I run try to run this directly on the `mysql` container using beekeeper. But it says syntax error during migration.

Reply:
The error you're getting is probably not from MySQL, but rather from SQLite which doesn't support Enums.

Reply:
Ah, that makes sense bhaia. In that case, should I just keep the migration as it is?

Reply:
Sure